packages/fxa-admin-server/src/gql/account/account.resolver.ts

```diff
@@ -182,6 +183,27 @@ export class AccountResolver {
     return account;
   }
 
+  @Features(AdminPanelFeature.AccountSearch)
+  @Query((returns) => [AccountType], { nullable: true })
+  @SentryTraced('accountByRecoveryPhone')
+  public async accountByRecoveryPhone(
+    @Args('phoneNumber', { nullable: false }) phoneNumber: string,
+    @Args('autoCompleted', { nullable: false }) autoCompleted: boolean,
+    @CurrentUser() user: string
+  ) {
+    this.eventLogging.onAccountSearch('phoneNumber', autoCompleted);
+    this.log.info('accountByRecoveryPhone', { phoneNumber, user });
+
+    let accounts = await this.db.account
+      .query()
+      .select(ACCOUNT_COLUMNS.map((c) => 'accounts.' + c))
+      .innerJoin('recoveryPhones', 'recoveryPhones.uid', 'accounts.uid')
+      .where('recoveryPhones.phoneNumber', phoneNumber)
+      .limit(50);
```

Comment:
(Since Dan assigned himself to review, I'm just walking by and casually looking through the windows...)

Why 50?

Reply:
It doesn't hurt performance.

But as for the actual number for the argument value, it make sense to have it match whatever the limit is for accounts per number.  Doesn't hurt anything the way you have it now though so 🤷.  I was mainly curious if there was a reason behind 50.